### PR TITLE
Fix Scene Object implementation consistency and Scene serialisation

### PR DESCRIPTION
**Pull request by tomvanmele**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Licini** [comment]
looking great, just curious why did you choose to complete drop the scene serialization test for IPY?

**Comment by tomvanmele** [comment]
> looking great, just curious why did you choose to complete drop the scene serialization test for IPY?

black, ruff, ... always add a comma after the last parameter of `VolMesh.__init__` (and any other class with a long parameter list). you can't turn this off, and IronPython can't deal with it.

this problem is only triggered in this particular test...

**Comment by Licini** [comment]
> > looking great, just curious why did you choose to complete drop the scene serialization test for IPY?
> 
> black, ruff, ... always add a comma after the last parameter of `VolMesh.__init__` (and any other class with a long parameter list). you can't turn this off, and IronPython can't deal with it.
> 
> this problem is only triggered in this particular test...

ok that's annoying 😅

**Comment by Licini** [comment]
@tomvanmele Is this one ready? I realized #1353 needs to go after this because there are a lot of overlaps

**Comment by tomvanmele** [comment]
yes i think it is ready

File: src/compas_rhino/scene/meshobject.py [hunk 1]
```diff
@@ -57,6 +53,9 @@ def __init__(self, mesh, disjoint=False, **kwargs):
         self._guids_facelabels = None
         self._guids_spheres = None
         self._guids_pipes = None
+        self.vertexgroup = vertexgroup
+        self.edgegroup = edgegroup
+        self.facegroup = facegroup
```
Comment (Licini) [inline src/compas_rhino/scene/meshobject.py:58]: Don't have to be this PR, but one thing to consider would be also allow to serialise the context specific parameters.

**Review by Licini — APPROVED** [review]
LGTM!

**Pull request merged**